src/API/WesternStatesWater.WestDaat.Accessors/ApplicationAccessor.cs

```diff
@@ -151,6 +153,40 @@ private async Task<ApplicationFindSequentialIdLoadResponse> FindSequentialDispla
         };
     }
 
+    private async Task<ApplicationDocumentLoadSingleResponse> GetApplicationDocument(ApplicationDocumentLoadSingleRequest request)
+    {
+        await using var db = _westDaatDatabaseContextFactory.Create();
+
+        var document = await db.WaterConservationApplicationDocuments
+            .AsNoTracking()
+            .Where(doc => doc.Id == request.WaterConservationApplicationDocumentId)
+            .ProjectTo<SupportingDocumentDetails>(DtoMapper.Configuration)
+            .FirstOrDefaultAsync();
```

Comment:
Eh - whenever we know there should be exactly one I like to use `Single`, so it blows up if there's multiple. Can sometimes help catch weird bugs instead of silently returning the first item

```suggestion
            .SingleOrDefaultAsync();
```